YatimaStdLib/Algebra/Defs.lean

```diff
@@ -0,0 +1,352 @@
+/-! 
+This file is largely a port of `Mathlib`'s `Algebra.Group.Defs`
+-/
```

Comment:
No copyright there?

Reply:
Sorry I don't know much about the conventions here, how should it be copyrighted? If you mean a header, it didn't have one

Reply:
Yeah I'm checking https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/Algebra/Group/Defs.lean

If it doesn't have a copyright then I guess we can just provide a permalink:
https://github.com/leanprover-community/mathlib4/blob/c46f8b6aa0f6f19a50a536da61eee3945d2e7266/Mathlib/Algebra/Group/Defs.lean